readme.md

```diff
@@ -28,6 +28,23 @@ If you're a developer, and want to get started and [contribute](#Contributing) t
 5) Run `CRender.exe` and enjoy!
 
 ***
-## Contributing
+## Building
+
+### Linux
+
+1) To build this project you will require [Intel Rendering Toolkit](https://software.intel.com/content/www/us/en/develop/tools/oneapi/rendering-toolkit/download.html) (online version), CMake, and `libxi-dev build-essential xorg libglfw3 libglfw3-dev libxinerama-dev libxcursor-dev` (these are the packages for Debian/Ubuntu)
```

Comment:
Specify you only need Embree 3.0 and OpenImageDenoise?

Reply:
Those have been added in.